jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/Party.java

```diff
@@ -29,7 +29,7 @@ public class Party {
   @Id
   @GeneratedValue
   @Column(name = "id", nullable = false)
```

Comment:
```suggestion
  @Column(name = "id", length = 100, nullable = false)
```

Would adding `length` to the annotation make sense when it's an ID?